### PR TITLE
Fields have been generated so we can uncommented those methods

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @sdroege @EPashkin

### CODE REVIEW

File: src/event_key.rs [hunk 1]
```diff
@@ -38,8 +39,7 @@ impl EventKey {
         self.as_ref().group
     }
 
-    // TODO: to be uncommented when the field will be generated
-    // pub fn get_is_modifier(&self) -> bool {
-    //     from_glib(self.as_ref().is_modifier)
-    // }
+    pub fn get_is_modifier(&self) -> bool {
+        self.as_ref().is_modifier != 0
```
Comment (EPashkin) [inline src/event_key.rs]: Its bitfield, so IMHO better something like `self.as_ref().is_modifier & 1 != 0`,
same for `is_stop`

**Comment by GuillaumeGomez** [comment]
Updated.

File: src/auto/versions.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-Generated by gir (https://github.com/gtk-rs/gir @ 914ca73)
+Generated by gir (https://github.com/gtk-rs/gir @ 1ae7210)
```
Comment (EPashkin) [inline src/auto/versions.txt]: This file changed but submodule not updated.

File: src/event_key.rs [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 use gdk_sys;
 use glib::translate::*;
+use libc;
```
Comment (EPashkin) [inline src/event_key.rs]: IMHO this use not used.

Comment (EPashkin) [reply src/event_key.rs]: https://travis-ci.org/gtk-rs/gdk/jobs/508079289#L673

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
Thanks, 👍

File: src/event_key.rs (hunk 3 above)
Comment (GuillaumeGomez) [reply src/event_key.rs]: No it wasn't. Weird that I added it... Good catch!

**Comment by sdroege** [comment]
:+1:

**Pull request merged**